configs/recognition/c3d/README.md

```diff
@@ -0,0 +1,74 @@
+# C3D
+
+## Introduction
+```
+@ARTICLE{2014arXiv1412.0767T,
+author = {Tran, Du and Bourdev, Lubomir and Fergus, Rob and Torresani, Lorenzo and Paluri, Manohar},
+title = "{Learning Spatiotemporal Features with 3D Convolutional Networks}",
+keywords = {Computer Science - Computer Vision and Pattern Recognition},
+year = 2014,
+month = dec,
+eid = {arXiv:1412.0767}
+}
+
+@article{Tran2014C3DGF,
+  title={C3D: Generic Features for Video Analysis},
+  author={D. Tran and Lubomir D. Bourdev and R. Fergus and L. Torresani and Manohar Paluri},
+  journal={ArXiv},
+  year={2014},
+  volume={abs/1412.0767}
+}
+```
+
+Before train or test the C3D model, you should download the [mean file](https://download.openmmlab.com/mmaction/recognition/c3d/c3d_train01_16_128_171_mean.npy), which is ported from [here](https://github.com/facebookarchive/C3D/blob/master/C3D-v1.0/examples/c3d_finetuning/train01_16_128_171_mean.binaryproto).
+
+## Model Zoo
+
+### UCF-101
+
+| config | resolution | gpus | backbone | pretrain | top1 acc | top5 acc | inference_time(video/s) | gpu_mem(M) | ckpt | log | json |
+|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
+|[c3d_sports1m_16x1x1_30e_ucf101_rgb.py](/configs/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb.py)|128x171|8| c3d | sports1m | 80.04(83.37) | 95.19(96.25) | x | 6053 | [ckpt](https://download.openmmlab.com/mmaction/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb/c3d_sports1m_16x1x1_30e_ucf101_rgb_20201016-e0d4bd99.pth)|[log](https://download.openmmlab.com/mmaction/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb/20201016_101722.log)|[json](https://download.openmmlab.com/mmaction/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb/20201016_101722.log.json)
+
+Notes:
+
+1. The author of C3D used SVM to classify videos on UCF-101, we simply used a linear classifier and the **accuracy in brackets** was tested under the TSN framework with 10 segments.
+2. The **gpus** indicates the number of gpu (32G V100) we used to get the checkpoint. It is noteworthy that the configs we provide are used for 8x4 gpus as default.
```

Comment:
"It is noteworthy that the configs we provide are used for 8x4 gpus as default." wrong, for C3D, it is 8 gpus as default, right ?